packages/contract_templates/partials/call.handlebars

```diff
@@ -1,3 +1,3 @@
-public {{this.name}} = {
+public {{this.tsName}} = {
```

Comment:
Why do we need this change?

Reply:
Just to be safe I added backwards compatibility for callbacks on older ERC721 and new ERC721 with operator by overloading the onERC721Received and returning the correct bytes4.

This is a view and needs our overloaded workaround  `onERC721Received` and `onERC721Received1`.